Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -117,35 +174,94 @@ private class CustomCell: UITableViewCell {
         title.alpha = 1
         setNeedsLayout()
     }
+    
+    @objc func SELdidPan(recognizer: UIPanGestureRecognizer!) {
+        switch (recognizer.state) {
+        case .Began:
+            self.startLocation = self.backgroundHolder.center;
+            break
```

Comment:
Nit: You don't need the `break` in Swift if there is at least one line of code in the `case`.